help/sources/tutorials/ui/create/analytics/mixpanel.md

```diff
@@ -0,0 +1,121 @@
+---
+keywords: Experience Platform;home;popular topics;sources;connectors;source connectors;sources sdk;sdk;SDK
+title: Self-service documentation template for the UI
+description: Learn how to create a Mixpanel source connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+# Create a *Mixpanel* source connection in the UI
+
+This tutorial provides steps for creating a *Mixpanel* source connector using the Platform user interface.
+
+## Overview
+
+[Mixpanel](https://www.mixpanel.com) is a product analytics tool that enables capturing data on how users interact with a digital product. Mixpanel allows you to analyze this product data with simple, interactive reports that let you query and visualize the data with just a few clicks.
+
+This Adobe Experience Platform [sources](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en) leverages the [Event Export API > Download](https://developer.mixpanel.com/reference/raw-event-export) to download your event data as it is received and stored within Mixpanel, along with all event properties (including distinct_id) and the exact timestamp the event was fired into Experience Platform.
```

Comment:
Let's rephrase the first part to: "The Mixpanel [source](link) leverages..."

Reply:
The 1st bullet would read odd, have updated [Event Export API > Download] to [Mixpanel Event Export API > Download]. The Mixpanel prefix was missed.